### PR TITLE
fix(css): update color-scheme and prefers-color-scheme documents

### LINKED ISSUE #35738: Using color-scheme to trigger a specific "prefers-color-scheme" media query
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme

### What specific section or headline is this issue about?

https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme#color_scheme_inheritance

### What information was incorrect, unhelpful, or incomplete?

In the provided demonstration, a default light color scheme style is applied, and completed with a dark color scheme style using the `prefers-color-scheme: dark` media query.
It shows that the `color-scheme: dark` attribute can be used to trigger the dark media query.

The problem is that it works as expected only if there is no `prefers-color-scheme: light` media query. If you add a "light" media query, it will still be preferred even if `color-scheme: dark` is set (supposing that the user prefers a light color scheme in their browser/OS settings).

### What did you expect to see?

More information about the scenario where both a `prefers-color-scheme: dark` and a `prefers-color-scheme: light` media query are set. In this scenario, it seems that user preferences will take the priority over the `color-scheme` attribute.

Also `matchMedia` could be mentioned: since it's a global function, it is not affected by `color-scheme` either, it will always reflect the user's preferences.

### Do you have any supporting links, references, or citations?

I've [crafted a codepen demo](https://codepen.io/ericburel/pen/ZEdmpNN).
I've explored this issue in the context of forcing a webcontainer dark or light mode using a toggle, for [tutorialkit](https://github.com/stackblitz/tutorialkit/issues/247#issuecomment-2328114730).

### Do you have anything more you want to share?

I initially suspected a bug but the behaviour seems similar in Chrome and Firefox. There has been [some discussions around forcing a specific color-scheme](https://github.com/mdn/content/issues?q=is%3Aissue+prefers-color-scheme+is%3Aclosed) but I don't think they covered this case.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/@media/prefers-color-scheme`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/@media/prefers-color-scheme/index.md
* Last commit: https://github.com/mdn/content/commit/4ecbac9e89961a132c1e7f5493ec94f60dcb1ee4
* Document last modified: 2024-08-08T16:03:20.000Z

</details>

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- fix https://github.com/mdn/content/issues/35738

### CODE REVIEW

File: files/en-us/web/css/color-scheme/index.md [hunk 1]
```diff
@@ -7,7 +7,14 @@ browser-compat: css.properties.color-scheme
 
 {{CSSRef}}
 
-The **`color-scheme`** [CSS](/en-US/docs/Web/CSS) property allows an element to indicate which color schemes it can comfortably be rendered in.
+The **`color-scheme`** [CSS](/en-US/docs/Web/CSS) property allows an element to indicate which color schemes it can comfortably be rendered in. User agents change the following aspects of the browser-controlled UI to match the used color scheme:
```
Comment (chrisdavidmills) [inline files/en-us/web/css/color-scheme/index.md:10]:
```suggestion
The **`color-scheme`** [CSS](/en-US/docs/Web/CSS) property allows an element to indicate which color schemes it can comfortably be rendered in. User agents change the following aspects of the UI chrome to match the used color scheme:
```

File: files/en-us/web/css/color-scheme/index.md [hunk 2]
```diff
@@ -7,7 +7,14 @@ browser-compat: css.properties.color-scheme
...
+
+- the color of the canvas surface
```
Comment (chrisdavidmills) [inline files/en-us/web/css/color-scheme/index.md:12]:
```suggestion
- The color of the canvas surface.
```

(Capitalize the other list items too, and add a period.)

File: files/en-us/web/css/color-scheme/index.md [hunk 3]
```diff
@@ -7,7 +7,14 @@ browser-compat: css.properties.color-scheme
...
+- the default colors of scrollbars and other interaction UI
+- the default colors of form controls
+- the default colors of other browser-provided UI, such as "spellcheck" underlines
+
+Component authors must use [`prefers-color-scheme`](/en-US/docs/Web/CSS/@media/prefers-color-scheme) media query to support the color schemes on reset of the elements.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/color-scheme/index.md:17]:
```suggestion
Component authors must use the [`prefers-color-scheme`](/en-US/docs/Web/CSS/@media/prefers-color-scheme) media feature to support the color schemes on reset of the elements.
```

Also, what do you mean by "on reset of the elements"? I'm not sure what it means, although I do have some good guesses. You might want to expand on this a bit to make it clearer.

File: files/en-us/web/css/color-scheme/index.md [hunk 4]
```diff
@@ -35,11 +42,11 @@ The `color-scheme` property's value must be one of the following keywords.
 ### Values
 
 - `normal`
-  - : Indicates that the element isn't aware of any color schemes, and so should be rendered using the browser's default color scheme.
+  - : If [page's supported color schems](/en-US/docs/Web/HTML/Element/meta/name#color-scheme) are set then the value indicates that the element supports the page's color schemes. If the page's color scheme is not set, the element does not support any color scheme.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/color-scheme/index.md:45]:
```suggestion
  - : Indicates that the element can be rendered using the page's [color scheme](/en-US/docs/Web/HTML/Element/meta/name#color-scheme) settings. If the page does not have a color scheme set, the element is rendered using the page's default color settings.
```

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
@OnkarRuikar these changes seem reasonable, but I made some suggestions to hopefully help clarify a few things. I'm not an expert on this particular area of CSS, but these suggestions seemed to make sense based on background reading.

File: files/en-us/web/css/color-scheme/index.md [hunk 5]
```diff
@@ -7,7 +7,14 @@ browser-compat: css.properties.color-scheme
 
 {{CSSRef}}
 
-The **`color-scheme`** [CSS](/en-US/docs/Web/CSS) property allows an element to indicate which color schemes it can comfortably be rendered in.
+The **`color-scheme`** [CSS](/en-US/docs/Web/CSS) property allows an element to indicate which color schemes it can comfortably be rendered in. User agents change the following aspects of the browser-controlled UI to match the used color scheme:
+
+- the color of the canvas surface
+- the default colors of scrollbars and other interaction UI
+- the default colors of form controls
+- the default colors of other browser-provided UI, such as "spellcheck" underlines
```
Comment (OnkarRuikar) [inline files/en-us/web/css/color-scheme/index.md:15]:
```suggestion
- The color of the canvas surface.
- The default colors of scrollbars and other interaction UI.
- The default colors of form controls.
- The default colors of other browser-provided UI, such as "spellcheck" underlines.
```

File: files/en-us/web/css/color-scheme/index.md (hunk 3 above)
Comment (OnkarRuikar) [reply files/en-us/web/css/color-scheme/index.md:17]:
```suggestion
Component authors must use the [`prefers-color-scheme`](/en-US/docs/Web/CSS/@media/prefers-color-scheme) media feature to support the color schemes on the rest of the elements.
```

File: files/en-us/web/css/color-scheme/index.md (hunk 2 above)
Comment (OnkarRuikar) [reply files/en-us/web/css/color-scheme/index.md]: done

**Review by chrisdavidmills — APPROVED** [review]
@OnkarRuikar looking good, approving!

**Pull request merged**